rasa/shared/core/training_data/story_reader/markdown_story_reader.py

```diff
@@ -241,6 +241,33 @@ def parse_e2e_message(line: Text) -> Message:
 
         return example
 
+    def _parse_message_e2e(self, message: Message, line_num: int) -> UserUttered:
```

Comment:
This method looks very similar to the `_parse_message()`. Can we combine them together and use the `if self.use_e2e:` to reflect the necessary differencies?

Reply:
i thought about that, but then they are taking different inputs, one is taking only a string and one the whole message.